NICE.Identity.Management/ClientApp/src/views/User/User.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+type PropsType = {
+	match: any;
```

Comment:
Please try to avoid `: any` type definitions. Here we should be able to do something like:

```
import { RouteComponentProps } from "react-router-dom";
type TParams = { id: number };
type UserProps = {
} & RouteComponentProps<TParams>;
export const User = (props: UserProps) => <div>
				<p>User: {props.match.params.id}</p>
			</div>;
```